src/test/java/tests/MilestoneAliasTests.java

```diff
@@ -160,6 +161,76 @@ public void replaceMilestoneAliasesTest() {
     }
 
     @Test
+    public void replaceMilestoneAliasesTestNoRelevantMilestone() {
```

Comment:
You need to rename all the variable names here. `relevant` is very confusing; why is `irrelevant` can still be the result of a test while `relevant` can be a non-result of a test. Also, it would be good if you can give a comment before the milestones and each `current` alias about what situation the tests are simulating.